lib/csv_import.rb

```diff
@@ -7,6 +7,10 @@ def csv_import(filepath)
     string = File.read(filepath)
     require 'csv'
 
+    # sanitize input
+    string = string.encode('UTF-8', 'binary',
+                           invalid: :replace, undef: :replace, replace: '')
```

Comment:
Ok great! Tbh, this was a super edge case that came up while I was trying to debug a whitespace issue, so I doubt we'll ever experience an invalid UTF-8 character, but better to be safe than sorry :stuck_out_tongue:.